src/renderers/shaders/ShaderLib.js

```diff
@@ -549,7 +551,6 @@ THREE.ShaderLib = {
 				THREE.ShaderChunk[ "metalnessmap_fragment" ],
 				THREE.ShaderChunk[ "normal_phong_fragment" ], // use phong chunk for now
 				THREE.ShaderChunk[ "lightmap_fragment" ],
-				THREE.ShaderChunk[ "hemilight_fragment" ],
```

Comment:
Right, I have to update aomap_fragment to work on indirectReflectLight.diffuse, as totalAmbientLight is now obsolete.  I'll do that.